### DISCUSSION TITLE
Improve dry-run debug output to better support local conf development

### DISCUSSION THREAD

**Discussion by JanoschDeurer** [renovatebot/renovate#27524] [Suggest an Idea]
### Tell us more.

As per the documentation the `--platform=local` option is for developing and testing renovate configuration files locally. This works great for finding obvious errors: Invalid json and other forms of invalid configuration. For more advanced problems the following info can be found in the debug log:

- Name of the dependecy (depName)
- What is the current version (currentValue)
- What is the version after the update
- What is the datasource for the update
- What is the update type (major, minor, pin ect.)
- What type of versioning is used

All of this is very helpful and I wanted to suggest, that it could be extended by adding the following information:

- Is there a schedule defined
- Would a merge request be created
  - Would it be automerged

**Comment by rarkins** [comment]
Created https://github.com/renovatebot/renovate/issues/27600
